consul/hooks/run

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash -xe
+
+exec 2>&1
+
+DEVMODE={{cfg.dev.mode}}
+
+if [ "$DEVMODE" = true ] ; then
+  exec consul agent -dev
+else
+  exec consul agent {{~#if cfg.website}} -ui {{~/if}} -server -data-dir={{cfg.server.data-dir}} -bootstrap-expect {{cfg.bootstrap.expect}} -client {{cfg.client.bind}}
```

Comment:
Which is better, inlining all possible command flags wrapped with `if` statements, or writing out a config file and loading it with `-config-*` flags?

```
  -config-file=foo         Path to a JSON file to read configuration from.
                           This can be specified multiple times.
  -config-dir=foo          Path to a directory to read configuration files
                           from. This will read every file ending in ".json"
                           as configuration in this directory in alphabetical
                           order. This can be specified multiple times.
```